### PR TITLE
changes in presentation_inheritance

### PR DESCRIPTION
**Pull request by Marcinati**

### CODE REVIEW

**Review by nauka-programowania-MA — APPROVED** [review]

File: module2/presentation_inheritance.md [hunk 1]
```diff
@@ -169,20 +167,6 @@ Załóżmy, że dodajemy klasę `Screen`. Klasa ta wyświetla na bieżąco inter
...
-}
-.col {
-    flex: 1;
-    margin: 1%;
-}
-</style>
-
-<div class="container">
-
-<div class="col">
```
Comment (ziobron) [inline module2/presentation_inheritance.md:184]: Why do you want it to be one column only? :)
It's not visible on GH I guess but it is in a presentation.

Comment (Marcinati) [reply module2/presentation_inheritance.md:184]: pr

Comment (Marcinati) [reply module2/presentation_inheritance.md:184]: html is visible on https://coders.school/edu/courses/obiektowosc-2-dziedziczenie-interfejsy-klasy-abstrakcyjne/lesson/dziedziczenie/

Comment (ziobron) [reply module2/presentation_inheritance.md:184]: Ok, I haven't seen it. I'll do this in another way. Let me close this PR and deliver changes on my own.

**Comment by ziobron** [comment]
My above commit 38c7f90 fixes this issues.

**Pull request closed without merging**